napari/layers/points/points.py

```diff
@@ -1006,15 +928,6 @@ def size(self, size: float | np.ndarray | list) -> None:
                 ) from e
             else:
                 self._size = np.mean(size, axis=1)
-                warnings.warn(
-                    trans._(
-                        'Since 0.4.18 point sizes must be isotropic; the average from each dimension will be'
-                        ' used instead. This will become an error in version 0.6.0.',
```

Comment:
I think this line along with line 995 | 917 indicates that the whole try/except/else block should be removed?

Reply:
I will take a look at this tomorrow. Thanks for all the reviews @psobolewskiPhD

Reply:
I agree with Peter's assessment. Well spotted, I think I would have missed it!

Reply:
@psobolewskiPhD @jni I'm not sure that I have full context here. Please rereview. Thanks.